data/filters/templates/youtube-shorts.yaml

```diff
@@ -23,6 +24,8 @@ template: |
   www.youtube.com##ytd-watch-next-secondary-results-renderer ytd-reel-shelf-renderer
   {{! Subscriptions in list mode, hide the whole section as subsequent videos from same channel are currently pushed in separate sections }}
```

Comment:
This section should - in my eyes - be removed as the goal of this rule is the same as the newly added `Hides Shorts section on the Subscriptions page`.